.kokoro/build.sh

```diff
@@ -23,6 +23,17 @@ cd ${scriptDir}/..
 # include common functions
 source ${scriptDir}/common.sh
 
+function setJava() {
+  export JAVA_HOME=$1
+  export PATH=${JAVA_HOME}/bin:$PATH
+}
+
+# units-java8 uses both JDK 11 and JDK 8. GraalVM dependencies require JDK 11 to
+# compile the classes touching GraalVM classes.
+if [ ! -z "${JAVA11_HOME}" ]; then
+  setJava "${JAVA11_HOME}"
```

Comment:
JAVA11_HOME is now available in the container image for Kokoro:

https://github.com/googleapis/java-spanner/pull/1870#discussion_r880579798